docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-offer-prices-feature-integration.md

```diff
@@ -17,16 +17,16 @@ To start feature integration, integrate the required features:
 
 | NAME | VERSION | INTEGRATION GUIDE |
 |-|-|-|
-| Marketplace Product Offer | {{page.version}} | [Marketplace Product Offer feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-product-offer-feature-integration.html) |
-| Prices | {{page.version}} |[Prices feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-product-offer-feature-integration.html) |
 | Spryker Core | {{page.version}} | [Spryker Core feature integration](https://documentation.spryker.com/docs/spryker-core-feature-integration) |
+| Prices | {{page.version}} |[Prices feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-product-offer-feature-integration.html) |
+| Marketplace Product Offer | {{page.version}} | [Marketplace Product Offer feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-product-offer-feature-integration.html) |
 
 
 ### 1) Install the required modules using Composer
 
 Install the required modules:
 ```bash
-composer require spryker-feature/marketplace-product-offer-prices --update-with-dependencies
+composer require spryker-feature/marketplace-product-offer-prices:"dev-master" --update-with-dependencies
```

Comment:
discussed with @gibmirmilch , lets have release tags for now

Reply:
fixed, but I think need to update Documentation Guides as well.